examples/el_farol/Readme.md

```diff
@@ -0,0 +1,23 @@
+# El Farol
+
+This folder contains an implementation of El Farol restaurant model. Agents (simulated restaurant customers) decide whether to go to the restaurant or not based on their memory and reward from previous trials. Implications from the model have been used to explain how individual decision-making affects overall performance and fluctuation.
+
+Besides, it demonstrates how to deploy a cognitive model (Instance-Based Learning) under the mesa environment. IBL model reflects the recency and frequency effect in decision-making with memory. IBL model deployed from the PyIBL package was used to replicate the research paper [1]. 
```

Comment:
OK, while this is more informative now, I still can't tell what the observable difference between the IBL and non-IBL version. Would be good to tell the user instead of having them to discover this by themselves.

Reply:
What I meant is that, what is the behavioral difference caused by having the recency and frequency effect implemented?

Reply:
Gotcha. I will add those.